docs/data-management/metrics/certified-metrics.md

```diff
@@ -29,27 +29,181 @@ metrics:
+| `aggregation_timeframe_unit` <br></br> (optional) | The time unit to use: `minutes`, `hours`, `days`, or `weeks` | `days` |
+| `winsorization_lower_percentile` <br></br> (optional) | Percentile at which to clip aggregated metrics | 0.001 |
+| `winsorization_upper_percentile` <br></br> (optional) | Percentile at which to clip aggregated metrics | 0.999 |
+| `filters` <br></br> (optional) | A list of filters to apply to metric, each containing a fact property, an operation (`equals` or `not_equals`), and a list of values | <pre><code>- fact_property: Source <br></br>  operation: equals <br></br>  values: <br></br>   - organic <br></br>   - search </code></pre>  |
+| `retention_threshold_days` <br></br> (optional, numerators only) | Number of days to use in retention calculation (only used if `operation` = `retention`) | 7 |
+| `conversion_threshold_days` <br></br> (optional, numerators only) | Number of days to use in conversion calculation (only used if `operation` = `conversion`) | 7 |
+| `threshold_metric_settings` <br></br> (optional, numerators only) | Setting for threshold metrics | <pre><code>comparison_operator: gt <br></br>aggregation_type: sum <br></br>breach_value: 0 <br></br>timeframe_unit: days <br></br>timeframe_value: 3 </code></pre> |
+
+
+*Note that `fact_name` can reference facts defined in a different yaml file.
+
+### Examples
+
+#### Simple Revenue Metrics
+
+As an example, if you have a central source-of-truth revenue table you can define both the table schema (`fact_source`) and metric semantics (`metrics`) in a yaml file like the one below. When Gross Revenue is analyzed in an experiment, Eppo will sum the `gross_revenue` column grouped by `user_id` and then measure the differences in these user-level aggregates between variants. To read more about how metrics are compiled into SQL, see the [Simple Metrics](/data-management/metrics/simple-metric) page.
+
+```yaml
+fact_sources:
+- name: Revenue
+  sql: |
+    SELECT user_id
+        ,  revenue_timestamp
+        ,  gross_revenue
+        ,  net_revenue 
+      FROM analytics_prod.revenue
+  timestamp_column: revenue_timestamp
+  entities:
+  - entity_name: User
+    column: user_id
+  facts:
+  - name: Gross Revenue
+    column: gross_revenue
+  - name: Net Revenue
+    column: net_revenue
+
+metrics:
+- name: Gross Revenue
+  entity: User 
+  numerator:
+    fact_name: Gross Revenue
+    operation: sum
+- name: Net Revenue
+  entity: User 
+  numerator:
+    fact_name: Net Revenue
+    operation: sum
+```
+
+#### Measuring User-Level Metrics for Company-Randomized Experiments
+
+In B2B businesses it's often impractical to randomize at the user level. Instead, experiments are typically randomized by company. It is however still valuable to understand any impacts on user-level behavior. Fortunately Eppo makes this easy through [Ratio Metrics](/data-management/metrics/ratio-metric).
```

Comment:
Let's remember to update this link to point to the clustered analysis guide once we merge https://github.com/Eppo-exp/eppo-docs/pull/325

Reply:
Yep definitely!